src/include/duckdb/common/enums/date_part_specifier.hpp

```diff
@@ -50,6 +50,63 @@ enum class DatePartSpecifier : uint8_t {
 	BEGIN_INVALID = INVALID,
 };
 
+inline string DatePartSpecifierToString(DatePartSpecifier date_part) {
```

Comment:
This shouldn't be necessary - you can just use `EnumUtil::ToString(date_part)` to convert to string

Reply:
Oh nice! I already felt like I was missing something. Will update, thanks :)